src/main/java/sirius/biz/jobs/params/Autocompleter.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.jobs.params;
+
+import sirius.db.mixing.Entity;
+import sirius.kernel.di.std.AutoRegister;
+import sirius.kernel.di.std.Named;
+import sirius.web.controller.AutocompleteHelper;
+import sirius.web.http.WebContext;
+
+import java.util.Collection;
+import java.util.function.Consumer;
+
+/**
+ * An abstract class that can be implemented to supply {@link AutocompleteHelper.Completion Completions} for entity-based token-autocomplete fields.
```

Comment:
Optional: Please begin JavaDoc with a verb describing what the interface does. The current JavaDoc is even false as it mentions an abstract class, but the documented code provides an interface instead.

Something like this would be my suggestion: `Allows to supply completions for entity-based token-autocomplete fields.`